lib/rules/focus.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "focus",
+  "selector": "[onFocus]",
+  "tags": [
+    "wcag2a",
+    "wcag211",
+    "cat.keyboard"
+  ],
+  "metadata": {
+    "description": "Ensure that executes code onFocus do not loose it when code is executed",
```

Comment:
This sentence is misleading.

Do you mean `Ensure that onFocus code upon execution does not lose focus`, or something similar?